src/common/FileSystem.cpp

```diff
@@ -1058,15 +1058,15 @@ static void InternalLoadPak(const PakInfo& pak, Util::optional<uint32_t> expecte
 
 	if (pak.type == pakType_t::PAK_ZIP) {
 		if (!isLegacy) {
-			fsLogs.Notice("Loading pak '%s'...", pak.path.c_str());
+			fsLogs.WithoutSuppression().Notice("Loading pak '%s'...", pak.path.c_str());
```

Comment:
nit: I prefer "ForcePrint()" or something to denote what it will do and not what it will not do.

Reply:
I'm not a fan of this suggestion since ForcePrint sounds like it should actually print something, while what the WithoutSuppression function actually does is only returning an altered logger object. Also, the mention of "suppression" makes it clear that it is related to the other functions with "suppression" in the name.

Reply:
sg